src/components/TodoList.tsx

```diff
@@ -0,0 +1,69 @@
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+import { TodoItem } from './TodoItem';
+
+type Props = {
+  todosToShow: Todo[],
+  tempTodo: Todo | null,
+  onRemoveTodo:(id: number) => void,
+  deletingTodoId: number,
+  isDeletingCompleted: boolean,
+  onChangeTitle: (todo: Todo, title: string) => void,
+  onToggleTodo: (todo: Todo) => void,
+};
+
+export const TodoList: React.FC<Props> = ({
+  todosToShow,
+  tempTodo,
+  onRemoveTodo,
+  deletingTodoId,
+  isDeletingCompleted,
+  onChangeTitle,
+  onToggleTodo,
+}) => {
+  return (
+    <section className="todoapp__main">
+      {todosToShow.map(todo => (
+        <TodoItem
+          key={todo.id}
+          todo={todo}
+          deletingTodoId={deletingTodoId}
+          onRemoveTodo={onRemoveTodo}
+          isDeletingCompleted={isDeletingCompleted}
+          onChangeTitle={onChangeTitle}
+          onToggleTodo={onToggleTodo}
+        />
+      ))}
+
+      {tempTodo && (
+        <div
```

Comment:
use `TodoItem` here as well
adjust `TodoItem` component to make it possible, for example: pass `isTodoChanging` as prop (calculate it inside `map` callback in the list)